src/test/java/de/uni_stuttgart/it_rex/course/written/service/CourseServiceExtendedTestIT.java

```diff
@@ -36,7 +36,7 @@ void updateNotExisting() {
     courseDTO.setMaxFoodSum(COURSE_MAX_FOOD_SUM);
 
     BadRequestAlertException e = assertThrows(BadRequestAlertException.class,
-        () -> courseResourceExtended.updateCourse(courseDTO));
+        () -> courseResourceExtended.patchCourse(courseDTO));
     assertThat(e.getMessage()).isEqualTo(EXPECTED_EXCEPTION_MESSAGE);
   }
 }
```

Comment:
updateNotExisting() -> patchNotExisting()

Reply:
The test should have been renamed. Fixed that